dot/sync/tip_syncer_integeration_test.go

```diff
@@ -208,18 +209,27 @@ func TestTipSyncer_handleTick_case2(t *testing.T) {
 			targetNumber: uintPtr(header.Number),
 			direction:    network.Ascending,
 			requestData:  network.RequestedDataBody + network.RequestedDataJustification,
-			pendingBlock: s.pendingBlocks.getBlock(header.Hash()),
+			pendingBlock: &pendingBlock{
+				hash:    header.Hash(),
+				number:  201,
+				header:  header,
+				clearAt: time.Time{},
+			},
 		},
 	}
 	w, err := s.handleTick()
 	require.NoError(t, err)
+	assert.Greater(t, w[0].pendingBlock.clearAt, time.Now())
```

Comment:
nit just to be safe and avoid panics

```suggestion
	require.NotEmpty(t, w)
	assert.Greater(t, w[0].pendingBlock.clearAt, time.Now())
```